recipes/libvault/all/conanfile.py

```diff
@@ -49,29 +61,18 @@ def layout(self):
 
     def requirements(self):
         # public header VaultClient.h includes curl/curl.h and use several functions
-        self.requires("libcurl/[>=7.78.0 <9]", transitive_headers=True, transitive_libs=True)
+        self.requires("libcurl/[>=7.78.0 <9]", transitive_headers=True)
```

Comment:
Just a minor question, could you give me a bit more insight into why this was removed? Wy is it not needed?

Reply:
I think it's because of this https://github.com/abedra/libvault/pull/27 So, there is no need to make it visible downstream anymore.